documentation/book/con-custom-resources-status.adoc

```diff
@@ -6,16 +6,53 @@
 
 = {ProductName} custom resource status
 
-The `status` property of a {ProductName}-specific custom resource publishes the _current_ state of the resource to users and tools that need the information.
+The `status` property of a {ProductName} custom resource publishes useful information about the resource to users and tools that need it.
 
-Status information is useful for tracking progress related to a resource achieving its _desired_ state, as defined by the `spec` property. The `status` provides the time and reason the state of the resource changed and details of events preventing or delaying the Operator from realizing the desired state.
+Several resources have a `status` property, as described in the following table.
+
+[cols="2*",options="header",stripes="none",separator=¦]
+|===
+
+¦{ProductName} Resource
+¦Publishes information on...
+
+m¦xref:type-KafkaStatus-reference[]
+¦The Kafka cluster.
+
+m¦xref:type-KafkaTopicStatus-reference[]
+¦Kafka topics and their partitions.
+
+m¦xref:type-KafkaConnectStatus-reference[]
+¦The Kafka Connect cluster, if deployed.
+
+m¦xref:type-KafkaConnectS2Istatus-reference[]
+¦The Kafka Connect cluster with Source-to-Image support, if deployed.
+
+m¦xref:type-KafkaMirrorMakerStatus-reference[]
+¦The Kafka MirrorMaker tool, if deployed.
+
+m¦xref:type-KafkaUserStatus-reference[]
+¦Kafka users.
+
+m¦xref:type-KafkaBridgeStatus-reference[]
+¦The {ProductName} Kafka Bridge, if deployed.
+
+|===
+
+The `status` property includes information on:
+
+* The _current state_ of the resource described in _status conditions_
+
+* The resource's last `observedGeneration` (the generation of the resource to which the status applies)
+
+* Other information specific to the resource, for example the REST API endpoint for Kafka Connect connectors.
+
+A resource's current state is useful for tracking progress related to the resource achieving its _desired state_, as defined by the `spec` property. The status conditions provide the time and reason the state of the resource changed and details of events preventing or delaying the Operator from realizing the resource's desired state.
```

Comment:
@laidan6000 should operator be capitalized? It typically isn't in other documentation I've seen.

Reply:
Good spot. We're not capitalizing operator. I'll change to lower-case.